attributes/nftables.rb

```diff
@@ -0,0 +1,22 @@
+default['firewall']['nftables']['defaults'][:policy] = {
```

Comment:
Let's not inter-mix `String` and `Symbol` keys despite the fact that the `Mash` makes it work either way